src/MPS.jl

```diff
@@ -0,0 +1,330 @@
+using Random
+using LinearAlgebra
+using Graphs
+using BijectiveDicts: BijectiveIdDict
+
+abstract type AbstractMPS <: AbstractAnsatz end
+
+mutable struct MPS <: AbstractMPS
+    const tn::Ansatz
+    form::Form
+end
+
+Ansatz(tn::MPS) = tn.tn
+
+Base.copy(x::MPS) = MPS(copy(Ansatz(x)), form(x))
+Base.similar(x::MPS) = MPS(similar(Ansatz(x)), form(x))
+Base.zero(x::MPS) = MPS(zero(Ansatz(x)), form(x))
+
+defaultorder(::Type{MPS}) = (:o, :l, :r)
+boundary(::MPS) = Open()
+form(tn::MPS) = tn.form
+
```

Comment:
""" Builds an MPS from a list of input tensors. Default ordering for the legs is (physical, virtual left, virtual right) - can be specified with the `order` keyword argument. 
Does/Does not? check for consistent dimension of input tensor legs
Does/Does not? bring to any canonical form 
"""